projects/packages/assets/src/class-assets.php

```diff
@@ -529,6 +531,11 @@ public static function wp_default_scripts_hook( $wp_scripts ) {
 		$wp_scripts->add( 'wp-jp-i18n-state', false, array( 'wp-deprecated', 'wp-jp-i18n-loader' ) );
 		$wp_scripts->add_inline_script( 'wp-jp-i18n-state', 'wp.deprecated( "wp-jp-i18n-state", { alternative: "wp-jp-i18n-loader" } );' );
 		$wp_scripts->add_inline_script( 'wp-jp-i18n-state', 'wp.jpI18nState = wp.jpI18nLoader.state;' );
+
+		// Register the React JSX runtime script - used as a polyfill until we can update JSX transforms. See https://github.com/Automattic/jetpack/issues/38424.
+		// Note: when removing this, also remove the inclusion of `react-jsx-runtime` from test_wp_default_scripts_hook where it is expected to be returned when the $mock calls the 'add' method.
```

Comment:
Let's add a `@todo` here mentioning WordPress 6.5, because that's usually the first thing we look for when preparing PRs like #38386. 🙂
```suggestion
		// @todo Remove this when we drop support for WordPress 6.5
		// Note: when removing this, also remove the inclusion of `react-jsx-runtime` from test_wp_default_scripts_hook where it is expected to be returned when the $mock calls the 'add' method.
```

Reply:
Thanks for the tip. I extended the comment a bit to mention the test so added it in a new commit: [ce92f21](https://github.com/Automattic/jetpack/pull/38428/commits/ce92f212609042604a13cb00a565e0089306fa31)